Makefile

```diff
@@ -0,0 +1,5 @@
+clean:
```

Comment:
should this be checked in ?

Reply:
Thought of adding more scripts in Makefile and use those scripts against CI run commands